driver/src/test/java/com/impossibl/postgres/jdbc/ServerErrorTest.java

```diff
@@ -105,6 +105,7 @@ public void testPrimaryKey() throws Exception {
       assertEquals("public", sqle.getSchema());
       assertEquals("testerr", sqle.getTable());
       assertEquals("testerr_pk", sqle.getConstraint());
+      assertEquals("Key (id)=(1) already exists.", sqle.getDetail());
```

Comment:
Is this going to be too specific to a server version?

Reply:
Maybe, don't know. Any ideas?